elyra/pipeline/processor_kfp.py

```diff
@@ -169,6 +79,126 @@ def cc_pipeline():
 
         return None
 
+    def export(self, pipeline):
+        if pipeline.file_type not in ["tgz", "tar.gz", "zip", "yaml", "yml", "py"]:
+            self.log.warn("Pipeline file type not recognized...defaulting to tar.gz")
+            pipeline_file_ext = "tar.gz"
```

Comment:
Let's use `DEFAULT_FILETYPE` here, both as a parameter in the warning and the value.